receiver/opencensusreceiver/config.go

```diff
@@ -44,6 +44,8 @@ type Config struct {
 
 // tlsCredentials holds the fields for TLS credentials
 // that are used for starting a server.
+// TODO(ccaraman): Add validation to check that these files exist at configuration loading time.
+//  Currently, these values aren't validated until the receiver is initialized.
```

Comment:
What does "initialized" mean? We have config "loading" phase, we have receiver "creation" using factory and we have receiver "starting" phase.

Reply:
In the code's term then it would be 'starting'. I'll update the language in the comment.